### PR TITLE
Change typeface to Nunito

### PR DESCRIPTION
**Pull request by jancborchardt**
Based on discussions we had, most recently in https://github.com/nextcloud/server/issues/11873

- :abc: Change typeface from Open Sans to Nunito → its rounded design works well with our branding, and it also gives a much friendlier feel. 😊
- :eyes: Move from Light/Semibold to Regular/Bold for better readability 
- :control_knobs: Change main text color from harsh `#000000` to `#222222`

screenshot from 2018-10-19 12-26-32

### CODE REVIEW

**Comment by skjnldsv** [comment]
I really like it as well!
Could we also rethink our headings at the same time? Since we're changig the font, so we can really have differences between h1,h2,h3 and h4? Pleaaaase @jancborchardt :see_no_evil: ;)

:tada:

**Comment by MorrisJobke** [comment]
It is far better to read with this on a full HD screen over here.

**Comment by karlitschek** [comment]
looks good 👍

**Comment by jancborchardt** [comment]
@skjnldsv that’s a different discussion – let’s work on things step by step. :)

@MorrisJobke @jospoortvliet @karlitschek could I have your approving review then? ;)

Others from @nextcloud/designers please also check it out!

**Comment by jancborchardt** [comment]
> What is wrong with ttf?

@MorrisJobke according to caniuse:

> WOFF: Compressed TrueType/OpenType font that contains information about the font's source.
> WOFF2: TrueType/OpenType font that provides better compression than WOFF 1.0.

So quicker loading times. Nunito Regular+Bold in ttf is 230 kB, whereas in woff2 (with latin extended even) it’s 77 kB.

But alas – different topic. :)

**Review by skjnldsv — APPROVED** [review]
Awesome!! Nice that the font loading is prioritised now :)

**Comment by jancborchardt** [comment]
@juliushaertl @karlitschek @MorrisJobke @jospoortvliet so can I have your approving reviews? ;) The +1 does not make the approval process happy. :)

**Review by karlitschek — APPROVED** [review]

**Review by MariusBluem — APPROVED** [review]

**Comment by skjnldsv** [comment]
@jancborchardt tests failing:

``` bash
There were 6 errors:

1) Test\Preview\BackgroundCleanupJobTest::testCleanupSystemCron
imagettftext(): Could not find/open font

/drone/src/github.com/nextcloud/server/lib/private/Preview/TXT.php:83
/drone/src/github.com/nextcloud/server/lib/private/Preview/GeneratorHelper.php:59
/drone/src/github.com/nextcloud/server/lib/private/Preview/Generator.php:194
/drone/src/github.com/nextcloud/server/lib/private/Preview/Generator.php:118
/drone/src/github.com/nextcloud/server/lib/private/PreviewManager.php:205
/drone/src/github.com/nextcloud/server/tests/lib/Preview/BackgroundCleanupJobTest.php:104
/drone/src/github.com/nextcloud/server/tests/lib/Preview/BackgroundCleanupJobTest.php:112

2) Test\Preview\BackgroundCleanupJobTest::testCleanupAjax
imagettftext(): Could not find/open font

/drone/src/github.com/nextcloud/server/lib/private/Preview/TXT.php:83
/drone/src/github.com/nextcloud/server/lib/private/Preview/GeneratorHelper.php:59
/drone/src/github.com/nextcloud/server/lib/private/Preview/Generator.php:194
/drone/src/github.com/nextcloud/server/lib/private/Preview/Generator.php:118
/drone/src/github.com/nextcloud/server/lib/private/PreviewManager.php:205
/drone/src/github.com/nextcloud/server/tests/lib/Preview/BackgroundCleanupJobTest.php:104
/drone/src/github.com/nextcloud/server/tests/lib/Preview/BackgroundCleanupJobTest.php:134

3) Test\Preview\TXTTest::testGetThumbnail with data set #0 (-72, -43)
imagettftext(): Could not find/open font

/drone/src/github.com/nextcloud/server/lib/private/Preview/TXT.php:83
/drone/src/github.com/nextcloud/server/tests/lib/Preview/Provider.php:139
/drone/src/github.com/nextcloud/server/tests/lib/Preview/Provider.php:103

4) Test\Preview\TXTTest::testGetThumbnail with data set #1 (73, 5)
imagettftext(): Could not find/open font

/drone/src/github.com/nextcloud/server/lib/private/Preview/TXT.php:83
/drone/src/github.com/nextcloud/server/tests/lib/Preview/Provider.php:139
/drone/src/github.com/nextcloud/server/tests/lib/Preview/Provider.php:103

5) Test\Preview\TXTTest::testGetThumbnail with data set #2 (-14, 91)
imagettftext(): Could not find/open font

/drone/src/github.com/nextcloud/server/lib/private/Preview/TXT.php:83
/drone/src/github.com/nextcloud/server/tests/lib/Preview/Provider.php:139
/drone/src/github.com/nextcloud/server/tests/lib/Preview/Provider.php:103

6) Test\Preview\TXTTest::testGetThumbnail with data set #3 (70, -39)
imagettftext(): Could not find/open font

/drone/src/github.com/nextcloud/server/lib/private/Preview/TXT.php:83
/drone/src/github.com/nextcloud/server/tests/lib/Preview/Provider.php:139
/drone/src/github.com/nextcloud/server/tests/lib/Preview/Provider.php:103

--
```

File: lib/private/Avatar.php [hunk 1]
```diff
@@ -62,15 +62,15 @@ class Avatar implements IAvatar {
 
 	/**
 	 * https://github.com/sebdesign/cap-height -- for 500px height
-	 * Open Sans cap-height is 0.72 and we want a 200px caps height size (0.4 letter-to-total-height ratio, 500*0.4=200). 200/0.72 = 278px.
+	 * Nunito cap-height is 0.72 and we want a 200px caps height size (0.4 letter-to-total-height ratio, 500*0.4=200). 200/0.72 = 278px.
```
Comment (skjnldsv) [inline lib/private/Avatar.php]: Nunito cap-height is 0.73

**Review by skjnldsv — CHANGES_REQUESTED** [review]

File: lib/private/Avatar.php (hunk 1 above)
Comment (jancborchardt) [reply lib/private/Avatar.php]: Really? I just checked with the linked tool and it said 0.7153…

Comment (jancborchardt) [reply lib/private/Avatar.php]: Also with your pen, it says:
- Chrome: 0.7153846416247677
- Firefox: 0.7175

Comment (skjnldsv) [reply lib/private/Avatar.php]: EDIT: Nope, it's 0.713, so it's fine to leave it at 0.72
https://codepen.io/skjnldsv/pen/PydLBK/

**Review by skjnldsv — APPROVED** [review]

**Review by jospoortvliet — APPROVED** [review]
Awesome! :+1:

**Comment by skjnldsv** [comment]
Failure unrelated: `sh: 1: kill: No such process`

**Review by MorrisJobke — APPROVED** [review]
Looks good 👍

**Pull request merged**